### ISSUE TITLE
suggestion: brew install kubernetes-helm to allow both versions

### ISSUE THREAD

**Issue by drnic** [helm/helm#6787]
Currently `brew install kubernetes-helm` installs v2.15.1. I worry that when v3 is cut that it will automatically stop providing v2 `helm` as an option.

Similarly, currently we cannot encourage users to try v3 betas via helm; but we could.

Could we update the `kubernetes-helm` homebrew recipe to support a `brew install kubernetes-helm@2` to get a `helm` that is v2.X, even when v3.0.0 is shipped?

Also, could we add v3 betas to the recipe so ppl can install and test them via `brew install kubernetes-helm@3`?

**Comment by bacongobbler** [comment]
Hey Nic.

The homebrew-helm formula is a community-owned project, so it's up to the community on how to proceed here. You'll have to reach out to the maintainers of that formula on what's the suggested steps for that project. The ubuntu snap project and the kubernetes-helm packages in the AUR are also owned by the community. We're only responsible for what's released on the github release page and what's on [get.helm.sh](https://helm.sh/blog/get-helm-sh/).

Hope this helps!

**Comment by karuppiah7890** [comment]
I agree @drnic . People would want easy ways to test betas and also would want to use any stable version - there would be two when v3 stable releases. And I think while shipping v3, we should also put notes saying they have to get 2to3 plugin to migrate from v2 to v3 if they were already using v2. But all these discussions have to happen within the maintainers of the formula/brew community and same for other package managers like @bacongobbler said. We can help with advices on what can be done I guess. And I have seen many many PRs for Homebrew, not sure how they maintain and merge things, it's large scale but you can discuss and also raise PRs after checking if someone has already started his conversation / raise a PR or both
